doc/ssh_config/ssh_config.md

```diff
@@ -161,6 +175,20 @@ module sonic-ssh-server {
                         }
                     }
 				}
+				leaf auto_logout {
+					description "inactivity timeoout (min unit)";
+					default 15;
+					type uint32 {

```

Comment:
uint32 -> uint16?

Reply:
Same here, prefer not to limit by u16 in case we will extend in future.